test/common/network/BUILD

```diff
@@ -131,6 +131,9 @@ envoy_cc_test(
 envoy_cc_test(
     name = "dns_impl_test",
     srcs = ["dns_impl_test.cc"],
+    args = [
+        "--runtime-feature-disable-for-tests=envoy.reloadable_features.use_apple_api_for_dns_lookups",
```

Comment:
Is this needed? Does this actually go through the creation function?

Reply:
Yes, otherwise it will default to the apple version if you run the test suite on a mac.

Reply:
OK I see the test goes through the dispatcher. It probably shouldn't with this new code, but that's fine, I would just add a comment so it's more clear why this needs to be set.